qemu/tests/cfg/qmp_command.cfg

```diff
@@ -186,3 +186,7 @@
         - qmp_query-target:
             qmp_cmd = "query-target"
             cmd_result_check = equal
+        - qmp_query-machines:
+            qmp_cmd = "query-machines"
+            cmd_result_check = "contain"
+            cmd_return_value = "['hotpluggable-cpus', 'name', 'cpu-max']"
```

Comment:
@yiqianwei i have some concern about `cmd_return_value`.
1. old version qemu-kvm seems do not have `hotpluggable-cpus`
2. These are some keys in the qmp out put, do we need to check the value ?
Thanks!

Reply:
@yanan-fu,
1.For old qemu-kvm version,do we still need to test ? 
2.We don't need to check the values.because the values are not fixed.we only focus on the keys that will not change.

Reply:
@yiqianwei  
1. RHEL7 is needed.
2. make sense for key only, but how about `numa-mem-supported`, `default-cpu-type`, `deprecated` ?

Reply:
@yanan-fu 
1. We can delete "hotpluggable-cpus" key.
2. The return result of qemu-kvm slow train does not support "numa-mem-supported","default-cpu-type","deprecated"